metron-platform/metron-common/README.md

```diff
@@ -229,21 +249,33 @@ Using parens such as: "foo" : "\<ok\>" requires escaping; "foo": "\'\<ok\>\'"
     * input - List
   * Returns: Last element of the list
 
-### `FILL_LEFT`
-  * Description: Fills or pads a given string with a given character, to a given length on the left
+### `HLLP_CARDINALITY`
+  * Description: Returns HyperLogLogPlus-estimated cardinality for this set
   * Input:
-    * input - string
-    * fill - the fill character
-    * len - the required length
-  * Returns: the filled string
+    * hyperLogLogPlus - the hllp set
+  * Returns: Long value representing the cardinality for this set
 
-### `FILL_RIGHT`
-  * Description: Fills or pads a given string with a given character, to a given length on the right
+### `HLLP_INIT`
+  * Description: Initializes the set
   * Input:
-    * input - string
-    * fill - the fill character string
-    * len - the required length
-  * Returns: Last element of the list
+    * p (required) - the precision value for the normal set
+    * sp - the precision value for the sparse set. If sp is not specified the sparse set will be disabled.
+  * Returns: A new HyperLogLogPlus set
+
+### `HLLP_MERGE`
+  * Description: Merge hllp sets together
+  * Input:
+    * hllp1 - first hllp set
+    * hllp2 - second hllp set
+    * hllpn - additional sets to merge
+  * Returns: A new merged HyperLogLogPlus estimator set
+
+### `HLLP_OFFER`
```

Comment:
I'm not in love with `HLLP_OFFER`..I'd prefer something like `HLLP_ADD` to match `BLOOM_ADD`.  I understand that HLLP is probabalistic, so they're probably being cagey about having their API calls make checks that their maths can't check, but I think we can educate sufficiently to make it safe to call this `ADD` IMO.